util/ranger/ranger_test.go

```diff
@@ -1267,14 +1257,11 @@ func TestCompIndexInExprCorrCol(t *testing.T) {
 		SQL    string
 		Result []string
 	}
-
-	testData, err := testdata.LoadTestSuiteData("testdata", "ranger_suite")
-	require.NoError(t, err)
 	testData.GetTestCases(t, &input, &output)
 	for i, tt := range input {
-		testData.OnRecord(func() {
+		testdata.OnRecord(func() {
```

Comment:
can this work? TestData should have no methond `OnRecord` now,  `OnRecord` is a function as your previous change

Reply:
Yes. `OnRecord` never uses the receiver and effectively a function more than a method.